python/desc/imsim/camera_readout.py

```diff
@@ -407,13 +471,13 @@ def set_itl_bboxes(amp):
         The updated AmpInfoRecord.
     """
     amp.setRawBBox(afwGeom.Box2I(afwGeom.Point2I(0, 0),
-                                 afwGeom.Extent2I(532, 2020)))
+                                 afwGeom.Extent2I(544, 2048)))
```

Comment:
Not for now: Could these geometries come from the camera object so that we could handle both E2V and ITL here?

Reply:
This function is only used by the `test_camera_readout.py` code, so I will move this function there.  These numbers pretty much have to be hardcoded like this (or at least gotten from some source other than `obs_lsstCam`) otherwise the test wouldn't be checking that the expected values were being used.